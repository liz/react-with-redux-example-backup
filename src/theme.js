// eslint-disable-next-line import/no-webpack-loader-syntax
import variables from 'sass-extract-loader?{"plugins" :["sass-extract-js"]}!./variables.scss';

export default variables;