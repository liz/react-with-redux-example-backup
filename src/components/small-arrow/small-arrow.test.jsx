import React from 'react';
import { mount } from 'enzyme';

import SmallArrow from './';
import theme from '../../theme';

describe('SmallArrow', () => {
    it('should match the snapshot', () => {
        const wrapper = mount(<SmallArrow />);

        expect(wrapper.html()).toMatchSnapshot();
    });

    it('renders SmallArrow with expected props', () => {
        const wrapper = mount(<SmallArrow />);

        expect(wrapper.find('SmallArrow')).toHaveLength(1);
        expect(wrapper.find('SmallArrow').props()).toEqual({
            color: theme.primaryColor,
            width: '5px',
        });
    });
});