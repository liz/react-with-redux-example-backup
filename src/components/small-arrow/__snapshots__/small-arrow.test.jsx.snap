// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SmallArrow should match the snapshot 1`] = `"<span class=\\"small-arrow__SmallArrowContainer-sc-7bica9-0 jKpmhL\\" width=\\"5px\\" color=\\"#d62027\\"></span>"`;
